content/projects/oop/animals/part1/_index.md

```diff
@@ -120,20 +120,18 @@ home.makeAllSounds() // this doesn't return anything
 home.adoptPet(dog1) // 1
 home.makeAllSounds()
 // this returns:
-// Bark
+// ["Bark"]
 
 home.adoptPet(cat) // 2
 home.makeAllSounds()
 // this returns:
-// Bark
-// Meow
+// ["Bark", "Meow"]
 
 home.adoptPet(dog2) // 3
 home.makeAllSounds()
 // this returns:
-// Bark
-// Meow
-// Bark
+// ["Bark", "Meow", Bark]
```

Comment:
Might have missed to add the last index in a string, you have Bark, belive should be "Bark"